python_modules/libraries/dagster-looker/dagster_looker/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1!0+dev"
```

Comment:
I believe right now if you drop the `version.py` file the release code will skip this module https://github.com/dagster-io/internal/blob/master/python_modules/automation-internal/automation_internal/release/dagster_module.py#L100 - that might be the easiest way to avoid publishing for now

Reply:
Makes sense. I removed that file for now. We can add it back once we're ready to publish.